.travis.yml

```diff
@@ -28,8 +50,9 @@ before_install:
   - travis_retry composer self-update
```

Comment:
this is no longer necessary, although it's not related to this PR I think you can take the opportunity to remove it